test/test_model_exports_to_core_aten.py

```diff
@@ -17,6 +19,7 @@ def _get_ops_list(m: torch.fx.GraphModule):
 
 
 class TestQuantizePT2EModels(TestCase):
+    @pytest.mark.xfail()
```

Comment:
This test was never run in CI because it lacks the`__main__` part below.  And it's broken atm, and I have created an issue to track that https://github.com/pytorch/pytorch/issues/113506